flask_ask/core.py

```diff
@@ -605,12 +623,15 @@ def _flask_view_func(self, *args, **kwargs):
 
     def _map_intent_to_view_func(self, intent):
         """Provides appropiate parameters to the intent functions."""
-        view_func = self._intent_view_funcs[intent.name]
-        argspec = inspect.getargspec(view_func)
-        arg_names = argspec.args
-        arg_values = self._map_params_to_view_args(intent.name, arg_names)
-
-        return partial(view_func, *arg_values)
+        try:
+            view_func = self._intent_view_funcs[intent.name]
+            argspec = inspect.getargspec(view_func)
+            arg_names = argspec.args
+            arg_values = self._map_params_to_view_args(intent.name, arg_names)
+            return partial(view_func, *arg_values)
+        except:
+            print("Error: Please add the ", intent.name, " to play to run correctly the skill.")
```

Comment:
I have understood the steps of flask-ask. The first reason for adding this **try..except** is because when someone creates an audio test if forgοt to add **ResumeIntent** or **PauseIntent** the error started from this point. So I thought if I added it here, I will prevent the user from seeing the error files and the user will see the forgotten Intents. if someone forgot the Intents that he adds in Intent schema he can see what missing.